cypress/integration/1_detectors.spec.js

```diff
@@ -207,7 +209,8 @@ describe('Detectors', () => {
 
     // Change detector name
     cy.get(`input[placeholder="Enter a name for the detector."]`)
-      .clearInput()
+      .wait(1000)
+      .ospClear()
       .focus()
       .realType('test detector edited');
```

Comment:
It seems the cypress tests are failing at this step; the name isn't being updated properly, so the subsequent tests fail to locate the test detector. I was able to resolve this locally by refactoring this section to the following:
```
// Change detector name
cy.get(`input[placeholder="Enter a name for the detector."]`)
  .type('{selectall}{backspace}test detector edited');

Reply:
@AWSHurneyt did not work, what did work was I added a click before typing into the field, so click, clear, and type worked.